ui/material_svg_icon.js

```diff
@@ -0,0 +1,42 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.ui.MaterialSVGIcon');
+
+goog.require('shaka.ui.Enums');
+goog.require('shaka.util.Dom');
+
+/**
+ * @final
+ * @export
+ */
+shaka.ui.MaterialSVGIcon = class {
+  /**
+   * @param {!HTMLElement} parent
+   */
+  constructor(parent) {
+    this.parent = parent;
+
+    /** @private {!SVGElement} */
+    this.svg_ = shaka.util.Dom.createSVGElement('svg');
+
+    /** @private {!SVGElement} */
+    this.path_ = shaka.util.Dom.createSVGElement('path');
+
+    this.svg_.classList.add('material-svg-icon');
+    this.svg_.setAttribute('viewBox', '0 -960 960 960');
+
+    this.svg_.appendChild(this.path_);
+    parent.appendChild(this.svg_);
+  }
+
+  /**
+   * @param {shaka.ui.Enums.MaterialDesignSVGIcons} icon
+   */
+  use(icon) {
```

Comment:
Please make sure this is exported so that people using custom buttons and menus can still use icons in the same way that shaka-player does. With the icon font it was easier because all icons were automatically available.

```suggestion
  /**
   * @param {shaka.ui.Enums.MaterialDesignSVGIcons} icon
   * @export
   */
  use(icon) {
```

Reply:
fixed that in commit [#45c9047](https://github.com/shaka-project/shaka-player/pull/8969/commits/45c90473427261aaafa37f4069b9cbd496fe08de). Thanks for pointing it out